pkg/object/mqttproxy/mqtt_test.go

```diff
@@ -1895,7 +1895,7 @@ func TestHTTPDeleteSession(t *testing.T) {
 	if err != nil {
 		t.Errorf("marshal http session %v failed, %v", data, err)
 	}
-	req, _ := http.NewRequest(http.MethodPost, "http://localhost:8888/delete/session", bytes.NewBuffer(jsonData))
+	req, _ := http.NewRequest(http.MethodDelete, "http://localhost:8888/delete/session", bytes.NewBuffer(jsonData))
```

Comment:
`http://localhost:8888/delete/session` is a typo?

Reply:
In test, path is different. In MQTTProxy, we registry by using Easegress api. In test, I just write these path bu their meaning. I will update it, make path in test and mqttproxy same.